pallene/checker.lua

```diff
@@ -349,14 +306,60 @@ function Checker:check_program(prog_ast)
                 end
             end
 
+        elseif group_kind == "Stat" then
+            -- skip
+
         else
             error("impossible")
         end
     end
 
+    local total_nodes = #tls
+    if total_nodes == 0 then
+        return prog_ast
```

Comment:
Should empty modules be allowed or should they be an error? If it were an error it would be one less corner case to worry about. What do you think?